### PR TITLE
feat(WEBRTC): improve react-native layout 

### PR DESCRIPTION
**Pull request by DeividVeloso**
Added navigation 
Changed layout to brand Telnyx

## 📝 To Do

- [x] All linters pass
- [x] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. cd examples/react-native
2. Connect your real device
3. Run npm run android.
4. See if it is with logos and colors about Telnyx.
5. Get your production credentials and try to make a call

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [x] Firefox
- [x] Safari

### CODE REVIEW

File: examples/react-native/screens/Credential.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    const {navigation} = this.props;
+    return navigation.navigate('dialer', {
+      username: this.state.username,
+      password: this.state.password,
+      production: this.state.production,
+    });
+  };
+
+  handleEyes = () => {
+    this.setState(prev => {
```
Comment (jonathan-telnyx) [inline examples/react-native/screens/Credential.js:27]: @DeividVeloso react-native doesn't support hooks?

File: examples/react-native/screens/Dialer.js [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+      });
+
+      this.client.on('telnyx.socket.open', () => {
+        console.log('Socket Open');
+      });
+      this.client.on('telnyx.socket.close', event => {
+        console.log('Socket Close', event);
+      });
+      this.client.on('telnyx.socket.error', error => {
+        console.log('error', error);
```
Comment (jonathan-telnyx) [inline examples/react-native/screens/Dialer.js:79]: Shouldn't it be `console.error`?

File: examples/react-native/screens/Dialer.js [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+    return this.state.connected ? 'Connected' : 'Disconnected';
+  }
+
+  _middle() {
+    if (this.state.call) {
+      let streamURL = null;
+      const {
+        options: {remoteStream = null, localStream = null},
+      } = this.state.call;
+      console.log('localStream', localStream, localStream.toURL());
```
Comment (jonathan-telnyx) [inline examples/react-native/screens/Dialer.js:172]: All those console.log should be present in the final version?

Comment (DeividVeloso) [reply examples/react-native/screens/Dialer.js:172]: It is only an example to user use as a base to build the own application

File: examples/react-native/screens/Dialer.js (hunk 2 above)
Comment (DeividVeloso) [reply examples/react-native/screens/Dialer.js:79]: If you use console.error in react native it crashes the app, I prefer to use console.log to not crash the app in dev mode.

File: examples/react-native/screens/Credential.js (hunk 1 above)
Comment (DeividVeloso) [reply examples/react-native/screens/Credential.js:27]: Support

**Pull request merged**